yaksh/xmlrpc_clients.py

```diff
@@ -62,7 +62,9 @@ def run_code(self, language, test_case_type, json_data, user_dir):
             server = self._get_server()
             result = server.check_code(language, test_case_type, json_data, user_dir)
         except ConnectionError:
-            result = json.dumps({'success': False, 'error': 'Unable to connect to any code servers!'})
+            result = json.dumps({'success': False,
+                'weightage': 0.0,
+                'error': 'Unable to connect to any code servers!'})
```

Comment:
Please describe what the return values are in the docstring very clearly.  This looks like the sum of the weights that the user code passed, correct?